exampleSite/content/encounters/uwu.md

```diff
@@ -0,0 +1,14 @@
+---
+title: The Weapon's Refrain
+card_image: /theme-assets/gaia-test.png
+tier_image: /theme-assets/gaia-test.png
```

Comment:
Not a complaint on this or the path, but do we want to make this optional in the editor and use `default` in hugo to provide one from theme assets for before we get photos for a fight? This way we can get a day 1 guide up without having specific media assets.

Reply:
Yeah I think making this optional for now would be good since we are only showing individual stuff at the moment

Reply:
Also, while I'm doing re-review, we probably need to get FE to generate a default image here so we can default it in hugo.